rj_gameplay/rj_gameplay/situation/decision_tree/plays.py

```diff
@@ -0,0 +1,168 @@
+"""This module contains all the plays used by the decision tree situation analyzer.
+Ported from the old situation alaysis."""
```

Comment:
Are these each supposed to contain a list of plays? That seems somewhat backwards. _All_ plays should provide an `is_applicable` method and then we should determine which plays we can use at runtime. Unless I'm misinterpreting this, it prevents us from dynamically selecting playbooks, which is a _very_ important feature.

Reply:
Nah, this is just a replacement for the enum we've been using before, but like it allows us to extend `ISituation` to in the future to require things for all situations, ie. "name" or "description" or whatever. This could be a big enum, but we'd need to do some metaclass magic to do the same thing. Both are identical approaches, I just thought that this was simpler.

Reply:
Sure, this is fine I guess. Could you leave a comment about the intentions (basically just an open enum)?